packages/flutter/test/widgets/title_test.dart

```diff
@@ -14,4 +14,13 @@ void main() {
     );
     expect(widget.toString, isNot(throwsException));
   });
+
+  testWidgets('should handle having no title', (WidgetTester tester) async {
+    final Title widget = new Title(
+      color: const Color(0xFF00FF00),
+      child: new Container(),
+    );
+    expect(widget.toString, isNot(throwsException));
+    expect(widget.title, equals(Title.kDefaultTitle));
+  });
```

Comment:
An even better test would be to provide a mock implementation of the backend for SystemChrome.setApplicationSwitcherDescription (there's test framework support for doing that) and making sure that that doesn't include the phrase "null".

Reply:
Done